massa-consensus-exports/src/channels.rs

```diff
@@ -7,17 +7,23 @@ use massa_protocol_exports::ProtocolCommandSender;
 
 use crate::events::ConsensusEvent;
 
-/// Contains a reference to the pool, selector and execution controller
-/// Contains a channel to send info to protocol
-/// Contains channels to send info to api
+/// Contains links to other modules of the node to be able to interact with them.
 #[derive(Clone)]
 pub struct ConsensusChannels {
+    /// Interface to interact with Execution module
     pub execution_controller: Box<dyn ExecutionController>,
+    /// Interface to interact with PoS module
     pub selector_controller: Box<dyn SelectorController>,
+    /// Interface to interact with Pool module
     pub pool_command_sender: Box<dyn PoolController>,
+    /// Channel use by the consensus to send instruction to the node globally
     pub controller_event_tx: crossbeam_channel::Sender<ConsensusEvent>,
+    /// Channel to send command to the protocol
     pub protocol_command_sender: ProtocolCommandSender,
+    /// Channel use by Websocket (if they are enable) to broadcast a new block integrated
     pub block_sender: tokio::sync::broadcast::Sender<Block>,
+    /// Channel use by Websocket (if they are enable) to broadcast a new block header integrated
     pub block_header_sender: tokio::sync::broadcast::Sender<BlockHeader>,
+    /// Channel use by Websocket (if they are enable) to broadcast a new block integrated
```

Comment:
I don't understand the diff with block_sender

Reply:
Me neither. @aoudiamoncef can you provide some context ?